packages/test/local-server-stress-tests/src/test/localServerStress.spec.ts

```diff
@@ -41,9 +41,7 @@ describe("Local Server Stress", () => {
 		// saveSuccesses,
 		configurations: { "Fluid.Container.enableOfflineLoad": true },
 		skip: [
-			...[46], // Number of keys not same
-			...[9, 79], // Number of subDirectories not same,
-			...[26], // Unexpected pending data for set/delete op
+			...[30, 69], // timeout
```

Comment:
does the test actually time out or is it just minimization? have you true un-commenting the above skip-minimization?